mini-cap/src/components/CondoRequestsView.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState } from 'react';
+import "../styling/CondoRequestsView.css";
+import PropTypes from "prop-types";
+import { MANAGEMENT_COMPANY } from "../backend/Constants";
+
+const CondoRequestsView = ({ role }) => {
+    const [currentStep, setCurrentStep] = useState(1);
+
+    useEffect(() => {
+    }, []);
+
+    const handleAdvance = () => {
+        if (currentStep < 4)
+            setCurrentStep(currentStep + 1);
+
+        // updateRequest(condoId, requestId);
```

Comment:
Go through the upateRequest method and see stepType. The return value while be the step its currently at